packages/orchestrator/main.go

```diff
@@ -442,6 +457,27 @@ func run(port, proxyPort uint) (success bool) {
 		return nil
 	})
 
+	g.Go(func() (err error) {
+		// this sets the error declared above so the function
+		// in the defer can check it.
+		hyperloopErr := hyperloopSrv.Start()
+		if hyperloopErr != nil {
+			hyperloopErr = fmt.Errorf("hyperloop server: %w", hyperloopErr)
+			zap.L().Error("hyperloop server error", zap.Error(hyperloopErr))
+
+			select {
+			case serviceError <- hyperloopErr:
```

Comment:
this will cause orchestrator to shutdown, not sure if hyperloop crash should do that